main.c

```diff
@@ -466,13 +477,36 @@ static void setup_property_changed_listener(void) {
 }
 #endif
 
+static void seat_handle_capabilities(void *data, struct wl_seat *seat,
+		uint32_t capabilities
+) {
+	struct seat *seat_tmp = data;
```

Comment:
Can we drop the `_tmp` suffix? (Applies to the whole patch)

Reply:
Sure, I dropped the suffix but still can't use "seat" since it's a global variable. Changed it to "self" in the seat handlers and "s" in the global handler, and left it as "seat_i" for iterating in main. Different suggestions?